src/annotations/core/AnnotationArea.cpp

```diff
@@ -327,6 +334,7 @@ void AnnotationArea::enableEditing()
     if(editableItem != nullptr) {
         mItemModifier->clear();
         editableItem->enableEditing();
+        itemsDeselected();
```

Comment:
This is required in order to clear the properties panel after 'Edit' is selected from the context menu.